test/Microsoft.AspNetCore.Mvc.FunctionalTests/RazorViewLocationSpecificationTest.cs

```diff
@@ -93,5 +94,20 @@ public RazorViewLocationSpecificationTest(MvcTestFixture<RazorWebSite.Startup> f
             // Assert
             Assert.Equal(expected, body.Trim(), ignoreLineEndingDifferences: true);
         }
+
+        [Fact]
+        public async Task PartialLayout_ThrowsIfRequiredSectionMissing()
+        {
+            // Arrange
+            var path = "http://localhost/PartialViewEngine/ViewPartialMissingSection";
+
+            //Act
+            var ex = await Assert.ThrowsAsync<InvalidOperationException>(() => Client.GetAsync(path));
+
+            Assert.Equal(
+                "The layout page '/Views/Shared/_PartialLayout.cshtml' cannot find the section 'section'"
+                + "in the content page '/Views/PartialViewEngine/PartialMissingSection.cshtml'.",
```

Comment:
Looks like there's a space missing at the end of the first line. Does this test somehow succeed?